v2/streaming-data-generator/src/main/java/com/google/cloud/teleport/v2/transforms/StreamingDataGeneratorWriteToGcs.java

```diff
@@ -70,21 +70,14 @@ public abstract static class Builder {
 
   /** A {@link DoFn} that converts UTF8 Bytes to JSON Object. */
   private static class Utf8BytesToJsonFn extends DoFn<byte[], String> {
-    private JsonParser jsonParser;
-
-    @Setup
-    public void setup() throws IOException {
-      jsonParser = new JsonParser();
-    }
-
     @ProcessElement
     public void processElement(
         @Element byte[] element,
         @Timestamp Instant timestamp,
         OutputReceiver<String> receiver,
-        ProcessContext context)
-        throws IOException {
-      receiver.output(jsonParser.parse(new String(element, StandardCharsets.UTF_8)).toString());
+        ProcessContext context) {
+      receiver.output(
+          JsonParser.parseString(new String(element, StandardCharsets.UTF_8)).toString());
```

Comment:
This construct seems weird. Why it needs to parse string to JSON and write back to String? Formatting, data validation? A comment would be helpful to understand if that's the case

Reply:
This is done for formatting reasons. Added comment to make it clear